src/components/ProjectIcon/index.js

```diff
@@ -1,26 +1,34 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { setDisplayName } from 'recompose'
+import DefaultProjectIcon from './DefaultProjectIcon'
 import './ProjectIcon.css'
 
 export const DefaultIcon = () => ''
 
-export const ProjectIcon = ({ name, size, ticker, className }) => {
+export const ProjectIcon = ({
+  name,
+  size,
+  ticker,
+  className,
+  useFallbackIcon
+}) => {
+  const Icon = useFallbackIcon ? DefaultProjectIcon : DefaultIcon
```

Comment:
@devanshu-santiment Why do we need to have useFallbackIcon. Can we understand do we have this icon or not inside this class?

Reply:
Had a conversation with Andriy and got to know that we will be using the same default icon for assets table as well. So I replaced that default Icon instead